src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationJenkinsGitlabTest.java

```diff
@@ -89,7 +90,13 @@ public void mockConnectorRequestsForSetup(ProgrammingExercise exercise) throws E
     }
 
     @Override
-    public List<Verifiable> mockConnectorRequestsForStartParticipation(ProgrammingExercise exercise, String username, Set<User> users) throws IOException, URISyntaxException {
+    public void mockCopyRepositoryForParticipation(ProgrammingExercise exercise, String username, HttpStatus status) throws URISyntaxException, IOException {
+        // TODO: implement
+    }
+
+    @Override
+    public List<Verifiable> mockConnectorRequestsForStartParticipation(ProgrammingExercise exercise, String username, Set<User> users, boolean ltiUserExists)
+            throws IOException, URISyntaxException {
```

Comment:
Same here: An empty method in an abstract class should be abstract instead

Reply:
This is intentional. This is a requirement by the interface that is implemented. This method has to be implemented later on, when we implement the tests for Jenkins